src/components/AdvertiserName/FollowUserButton/FollowUserButton.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { ErrorModal } from '@/components/Modals';
+import { OnboardingTooltip } from '@/components/OnboardingTooltip';
```

Comment:
```suggestion
import { OnboardingTooltip } from '@/components';
```

shouldn't it be this?

Reply:
ah yeah ill remove it